### ISSUE TITLE
MOODS dependency not referenced

### ISSUE THREAD

**Issue by ebirn** [CostaLab/reg-gen#107]
MOODS module is used, but no reference in requirements.txt or setup.py/setup.cfg
https://github.com/CostaLab/reg-gen/blob/79d5b2cfb33c272729e46f155714e42294fd4c10/rgt/motifanalysis/Motif.py#L13

only reference seen in .travis.yml, call to rgt-motifanalysis script fails
```
login0 [~]> rgt-motifanalysis 
Traceback (most recent call last):
  File "/software/RGT/0.11.7-foss-2018b-Python-2.7.15/bin/rgt-motifanalysis", line 11, in <module>
    load_entry_point('RGT==0.11.7', 'console_scripts', 'rgt-motifanalysis')()
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 476, in load_entry_point
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2700, in load_entry_point
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2318, in load
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 2324, in resolve
  File "/software/RGT/0.11.7-foss-2018b-Python-2.7.15/lib/python2.7/site-packages/rgt/motifanalysis/Main.py", line 16, in <module>
    import Match
  File "/software/RGT/0.11.7-foss-2018b-Python-2.7.15/lib/python2.7/site-packages/rgt/motifanalysis/Match.py", line 22, in <module>
    from .Motif import Motif, ThresholdTable
  File "/software/RGT/0.11.7-foss-2018b-Python-2.7.15/lib/python2.7/site-packages/rgt/motifanalysis/Motif.py", line 13, in <module>
    from MOODS import tools, parsers
ImportError: No module named MOODS
```

Best,
Erich

**Comment by fabio-t** [comment]
This should actually be fixed in setup.py, yes. It's not available on **pypi** yet (I've asked the author a long time ago), and on our instructions it's stated you should install it if you wish to use motif analysis (which is also what you see in the travis config file):

```
pip install https://github.com/fabioticconi/MOODS/tarball/pypi-ready
```

We'll see if we can put a fix in for the next release.

**Comment by fabio-t** [comment]
Fixed via 71e3f4fe6825c83e0e48179f21abe67651e4156c
